ui/src/app/model/conv.ts

```diff
@@ -162,13 +162,7 @@ export interface IInterleaveStatus {
 
 export interface IPrimaryKey {
   TableId: string
-  Columns: IPkColumnDefs[]
-}
-
-export interface IPkColumnDefs {
```

Comment:
where has this moved to ?

Reply:
The IPkColumnDefs[] interface was same as IIndexKey[] so removed it